docs/spelling_wordlist.txt

```diff
@@ -339,6 +339,9 @@ dask
 dat
 Databricks
 databricks
+DatabricksPartitionSensor
+DatabricksSqlSensor
+DatabricksTableChangesSensor
```

Comment:
These are probably not necessary. Use code double quotes (like we do in other places and the doc build will pass fine)

Reply:
Removed it, the local doc build succeeded.